web-ng/src/app/shared/converters/firebase-data-converter.ts

```diff
@@ -158,12 +162,23 @@ export class FirebaseDataConverter {
   }
 
   public static featureToJS(feature: Feature): {} {
-    const { layerId, location } = feature;
-    return {
-      // TODO: Set audit info (created / last modified user and timestamp).
-      layerId,
-      location,
-    };
+    // TODO: Set audit info (created / last modified user and timestamp).
+    if (feature instanceof LocationFeature) {
+      const { layerId, location } = feature;
+      return {
+        layerId,
+        location,
+      };
+    } else if (feature instanceof GeoJsonFeature) {
+      const { layerId, geoJson } = feature;
+      return {
+        layerId,
+        geoJson,
+      };
+    } else {
+      const { layerId } = feature;
```

Comment:
If the application itself is generating features of unknown types, we should probably throw an exception here. Wdyt?

Reply:
Done.